mesos-docker/run/run.sh

```diff
@@ -122,11 +122,37 @@ function check_if_container_is_up {
   fi
 }
 
+function quote_if_non_empty {
+
+  if [[ -n $1 ]];then
+    echo "\"$1\""
+  else
+    echo "$1"
+  fi
+
+}
+
+function check_mesos_version {
+
+  MIT_MESOS_HOST_VERSION=$(dpkg -s mesos | grep Version | awk '{print $2}')
+  MIT_MESOS_LIB_DOCKER_VERSION=$(docker exec $1 sh -c "dpkg -s mesos | grep Version" |  awk '{print $2}')\
+
+  if [[ ! "MIT_MESOS_HOST_VERSION" == "MIT_MESOS_LIB_DOCKER_VERSION" ]]; then
+    printMsg "WARN: Host and Docker images have different versions, Host:$MIT_MESOS_HOST_VERSION, Image: $MIT_MESOS_LIB_DOCKER_VERSION (image always reflects the latest version). Pls upgrade host."
```

Comment:
Is having different versions a real problem (-> test won't work), or a just a potential problem in some cases (-> should work most of the time, but may fail)?

If it is a real problem, we may want to just abord the execution.